test_container/tests/test/python3-cuda-flavor/pytorch.py

```diff
@@ -60,12 +61,13 @@ def forward(self, x):
                     with torch.no_grad():
                         y_pred = model(x_train)
                         mse = criterion(y_pred, y_train)
-                        return f'Mean Squared Error: {mse.item():.4f}'
+                        return mse.item()
                 /
                 '''))
 
         row = self.query(f"SELECT pytorchbasic.test_pytorch(1000);")[0]
-        self.assertIn('Mean Squared Error', row[0])
+
+        self.assertFalse(math.isnan(row[0]))
```

Comment:
I think, NaN is transformed to NULL in the DB

Reply:
Appearantly the build failed when fetching micromamba. Running the test locally , the assert is correct.